src/utils/i18n.utils.ts

```diff
@@ -0,0 +1,17 @@
+// TODO: replace all this functionality with i18n module
+export const makeRequiredErrorMessage = (fieldName: string) => `"${fieldName}" is a required field`;
+
+const ordinalFormatter = new Intl.PluralRules('en-US', {
+  type: 'ordinal'
+});
+
+const suffixes: Record<Intl.LDMLPluralRule, string> = {
+  one: 'st',
+  two: 'nd',
+  few: 'rd',
+  other: 'th',
+  zero: 'th',
+  many: 'th'
+};
+
+export const formatOrdinalNumber = (n: number) => `${n}${suffixes[ordinalFormatter.select(n)]}`;
```

Comment:
Neither `makeRequiredErrorMessage` nor `formatOrdinalNumber` relates to `i18n`.   
Please, split them by files with correct names

Reply:
All of this stuff will be replaced with i18n library, so let's leave it all here to prevent forgetting